tools/eslint-rules/restrict-let-in-for-loops.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview Prohibit the use of `let` in `for` loops.
```

Comment:
Nit: Maybe specify that it's just the declarations (or whatever the right terminology is)?  I think this makes it sound like we're forbidding it in the body as well, but we're not.